docs/tezos-client.md

```diff
@@ -0,0 +1,77 @@
+---
+id: tezos-client
+title: Setup tezos-client to work with Signatory
+---
+
+The `tezos-client` is the official Octez CLI tool. This document shows how to configure a new account in `tezos-client` that Signatory backs.
+
+`tezos-client` helps with verifying the setup and fills a need when operators of a baker require to transfer tokens out of their baking account to, for example, a hot wallet.
+
+## Getting tezos-client
+
+Download the latest `tezos-client` binary from the _Releases_ page at https://github.com/serokell/tezos-packaging 
+Make sure to choose the appropriate binary for your operating system and architecture.
+
+The `tezos-client` is also included `tezos-client` in the official Octez docker images.
```

Comment:
Is the second `tezos client` supposed to be there?

Reply:
ok